srcpkgs/podman/template

```diff
@@ -9,7 +9,7 @@ go_build_tags="seccomp apparmor containers_image_ostree_stub
  btrfs_noversion exclude_graphdriver_btrfs"
 hostmakedepends="pkg-config go-md2man"
 makedepends="gpgme-devel libseccomp-devel device-mapper-devel"
-depends="runc conmon cni-plugins slirp4netns containers.image"
+depends="virtual?oci-runtime conmon cni-plugins slirp4netns containers.image"
```

Comment:
You need to bump revision when changing metadata like dependencies, provides, etc. Otherwise the binary package will not be rebuilt.

Also you may be able to get around this virtual dependency by leaving it as is then using `ignorepkg=runc` in an xbps config file.